src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -76,16 +76,22 @@ public static Project GetProject(IVsHierarchy hierarchy)
             {
                 return project as Project;
             }
+
             return null;
         }
 
-        public IVsHierarchy GetIVsHierarchy(Project Project)
+        public IVsHierarchy GetIVsHierarchy(Project project)
```

Comment:
To to confirm, the only product code caller is `ProjectSystemFilter.IsAccepted` which checks is the `EnvDTE.Project` is `null` itself already (so shouldn't impact any consumers).